src/lwc/utilTemplateBuilder/utilTemplateBuilder.js

```diff
@@ -1,6 +1,51 @@
 /* eslint-disable @lwc/lwc/no-async-operation */
 import { ShowToastEvent } from 'lightning/platformShowToastEvent'
 
+const FORMAT_VERSION = '1.0';
+const ADVANCED_MAPPING = 'Data Import Field Mapping';
+const DEFAULT_FIELD_MAPPING_SET = 'Migrated_Custom_Field_Mapping_Set';
+const LANDING_PAGE_TAB_NAME = 'GE_Templates';
+const SORTED_BY = 'required';
+const SORT_ORDER = 'desc';
+const PICKLIST = 'Picklist';
+const NEW = 'new';
+const EDIT = 'edit';
+
+const PROP_API_NAME = 'apiName';
+const PROP_BATCH_HEADER_TAB_ERROR = 'hasBatchHeaderTabError';
+
+const EVENT_UPDATE_VALIDITY = 'updatevalidity';
+const EVENT_BATCH_HEADER_FIELD_UPDATE = 'updatebatchheaderfield';
+const EVENT_BATCH_HEADER_FIELD_UP = 'batchheaderfieldup';
+const EVENT_BATCH_HEADER_FIELD_DOWN = 'batchheaderfielddown';
+const EVENT_BATCH_HEADER_FIELD_ADD = 'addbatchheaderfield';
+const EVENT_BATCH_HEADER_FIELD_REMOVE = 'removebatchheaderfield';
+
+const OBJECT = 'object';
+const FUNCTION = 'function';
+
+const ADDITIONAL_REQUIRED_BATCH_HEADER_FIELDS = [
+    'Name'
+];
+Object.freeze(ADDITIONAL_REQUIRED_BATCH_HEADER_FIELDS);
+
+// We've opted to exclude the following batch fields related to
+// matching logic as we're removing the matching options page
+// from this flow.
+// We're considering putting matching options in a 'global
+// batch settings' area. Potentially in NPSP Settings.
+const EXCLUDED_BATCH_HEADER_FIELDS = [
```

Comment:
See comment above re: importing these so we can reference their `fieldApiName` properties and avoid explicit handling of namespaces.